### PR TITLE
terminal for ipython notebook

### PR DESCRIPTION
**Pull request by cekees**

### CODE REVIEW

File: pkgs/terminado.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+extends: [distutils_package]
+
+sources:
+  - url: https://github.com/takluyver/terminado.git
+    key: git:d54c7b9e750bcceeca8806622fbf8177edb4ab22
```
Comment (certik) [inline pkgs/terminado.yaml]: You should put the `key` first, `url` second.

**Comment by certik** [comment]
Otherwise looks good. The errors are unrelated.

**Comment by certik** [comment]
Shouldn't it depend on tornado?

**Comment by cekees** [comment]
Thanks @certik .

**Comment by certik** [comment]
I didn't test it, but otherwise it looks good. If it works for you, you can merge it.

**Comment by ahmadia** [comment]
This is fine to merge.

**Comment by jcftang** [comment]
Since we have a few ok's on this, let's merge it.

**Pull request merged**